api/src/main/java/marquez/db/mappers/DatasetVersionMapper.java

```diff
@@ -65,7 +66,8 @@ public DatasetVersion map(@NonNull ResultSet results, @NonNull StatementContext
               toFields(results, "fields"),
               columnNames.contains("tags") ? toTags(results, "tags") : null,
               stringOrNull(results, Columns.DESCRIPTION),
-              null);
+              null,
+              toFacetsOrNull(results));
```

Comment:
Can we use `Optional<Facet>`? Looks like Jackson has support for it in the `jackson-datatype-jdk8` package

Reply:
We can begin using optionals as parameters or drop them all together. I'll open an issue about uses of optionals in the code.